rpc/backend/blocks.go

```diff
@@ -46,10 +45,6 @@ func (b *Backend) BlockNumber() (hexutil.Uint64, error) {
 		return 0, fmt.Errorf("failed to parse block height: %w", err)
 	}
 
-	if height > math.MaxInt64 {
```

Comment:
should this have been removed?

Reply:
yes https://github.com/cosmos/evm-internal/pull/3